src/plugins/plugins/permissions.js

```diff
@@ -33,7 +34,7 @@ export class Permissions extends ChatPlugin {
     // Validate the password - if there is one.
     if (this.adminPassword && adminPassword === this.adminPassword) {
       const id = Permissions.nameToId(message.user.id);
-      this.bot.db.set(`permissions.users.${id}.roles.admin`, true).value();
+      this.bot.users.botUsers[id].roles.admin = true;
```

Comment:
Instead of modifying `this.bot.users` directly, here and below, it may be better to have a function for it:

`this.bot.addRole(id, 'admin')`